s2n.mk

```diff
@@ -116,6 +116,10 @@ ifdef S2N_DEBUG
 	CFLAGS += ${DEBUG_CFLAGS}
 endif
 
+ifdef S2N_GDB
+	CFLAGS += ${DEBUG_CFLAGS} -Og
```

Comment:
Is the `-Og` flag available on the really old gcc compilers we support in brazil? It seems to be introduced in gcc 4.8.